ossdbtoolsservice/driver/types/mysql_driver.py

```diff
@@ -297,8 +296,8 @@ def execute_dict(self, query: str, params=None):
                 if self.autocommit:
                     self._conn.commit()
                 return col_names, rows
-            except Exception:
-                return False
+            except mysql.connector.Error as e:
+                self.handle_connection_error(e)
```

Comment:
This will throw exception. 
Behaviour change from returning bool to throwing exception.
Validate this behaviour change is not affecting the existing usecases.

Reply:
reverted